media/css/careers/styles.scss

```diff
@@ -0,0 +1,45 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$image-path: '/media/protocol/img';
+
+@use './modules/hero-text.scss';
+@use './modules/hero-video.scss';
+@use './modules/home.scss';
+@use './modules/testimonials.scss';
+@use './modules/diversity.scss';
+@use './modules/locations.scss';
+@use './modules/job-listings.scss';
+@use './modules/position.scss';
+@use './modules/bottom.scss';
```

Comment:
I couple **Nits** here:
- I don't think that every page in careers needs to use all of these files, right? If they don't, I think we should should separate them into different bundles. (For instance, listings.html will need `job-listings.scss` but not `diversity.scss`) Would be interested to hear why that might be a good idea for performance, but it would make sense to my brain
- I think we should rename the `module` folder to `components`since that is a pattern that we use in other places in bedrock, but `modules` is a naming convention that we don't use in other places

Reply:
I _personally_ think it's fine that we're structuring it this way? We usually code out our stylesheets in this manner most of the time?